### ISSUE TITLE
Support for Textarea

### ISSUE THREAD

**Issue by stefanstelzer** [ng2-ui/ng2-auto-complete#55]
Hey guys,

is there any support forseen for textarea auto-complete support or should I contribute to the project?

Best Regards,

Stefan

**Comment by allenhwkim** [comment]
I don't picture this. Could you explain more about it?

**Comment by stefanstelzer** [comment]
I do have a texatarea that can be expanded by the user. This textarea should have the autocomplete ability as well. Now?

**Comment by allenhwkim** [comment]
Textarea is very different from input (mainly singleline vs multiline), and ng2-auto-complete does not have support for textarea at the moment.

I would suggest that you still can use this `ng2-auto-complete` as it is for your textarea  to display anywhere within your textarea because it has its own input field and dropdown.

I can picture this.

![image](https://cloud.githubusercontent.com/assets/1437734/20277243/c735e5cc-aa6d-11e6-9ab6-54a15788fe14.png)

There are two UI parts for this module. 
  1. Ng2AutoCompleteDirective
  2. Ng2AutoCompleteComponent

I have designed this to be used for multiple auto complete, and Ng2AutoCompleteComponent is the core of this. You can forget about Ng2AutoCompleteDirective, and just use Ng2AutoCompleteComponent for your own textarea autocomplete. e.g. `Ctrl-Tab` to show Ng2AutoCompleteComponent.

I may go too far.

Or, you can create Ng2AutoCompleteTextareaDirective by cloning Ng2AutoCompleteDirective.

The bottom line is, not having textarea support yet.
